packages/astro/src/assets/fonts/constants.ts

```diff
@@ -1,4 +1,18 @@
-import { GOOGLE_PROVIDER_NAME } from "./providers/google.js";
-import { LOCAL_PROVIDER_NAME } from "./providers/local.js";
+import { GOOGLE_PROVIDER_NAME } from './providers/google.js';
+import { LOCAL_PROVIDER_NAME } from './providers/local.js';
+import type * as unifont from 'unifont';
 
 export const BUILTIN_PROVIDERS = [GOOGLE_PROVIDER_NAME, LOCAL_PROVIDER_NAME] as const;
+
+export const DEFAULTS: unifont.ResolveFontOptions = {
+	weights: ['400'],
+	styles: ['normal', 'italic'],
+	subsets: ['cyrillic-ext', 'cyrillic', 'greek-ext', 'greek', 'vietnamese', 'latin-ext', 'latin'],
+	fallbacks: undefined,
+};
+
+export const VIRTUAL_MODULE_ID = 'virtual:astro:assets/fonts/internal';
+export const RESOLVED_VIRTUAL_MODULE_ID = '\0' + VIRTUAL_MODULE_ID;
+
+export const URL_PREFIX = '/_fonts/';
```

Comment:
Some considerations:
- this constant is used as a real browser URLs and file system. If possible I would avoid this approach and use two different constants. This is the naming convention (`_image`, etc.) is used for our internal endpoints, but we don't use it for internal directories
- this constant is used in browser URLs, which are affected by the `trailingSlash` option. I don't know yet if this endpoint will be called in SSR too, but I would avoid the trailing slash here, and add it later based on the configuration

Reply:
- Yeah I thought about it then forgot to update, wdyt about `/_astro/fonts`?
- I think we don't need to take the the trailing slash into account because it will always look like `{prefix}/name.{woff/ttf/etc}` (static asset)

Reply:
> I think we don't need to take the the trailing slash into account because it will always look like

Good to know, you might want to comment this somewhere as a reminder